content/docs/thinking-in-react.md

```diff
@@ -64,6 +64,17 @@ React 眾多的優點之ㄧ是它讓你能在寫程式的同時去思考你的
 <p data-height="600" data-theme-id="0" data-slug-hash="BwWzwm" data-default-tab="js" data-user="lacker" data-embed-version="2" class="codepen">在 <a href="http://codepen.io">CodePen</a> 中看<a href="https://codepen.io/gaearon/pen/BwWzwm">用 React 思考：第二步</a>。</p>
 <script async src="https://production-assets.codepen.io/assets/embed/ei.js"></script>
 
+在你有了 component 層級後，就可以開始實作你的應用程式了。最簡單的方式是為你的應用程式建立一個接收數據模型、render UI 且沒有互動性的版本。建立一個靜態版本需要打很多字，但不需要想很多，而加上互動性則相反，需要做很多的思考，打很少的字，所以最好的方式是把這幾個過程都分開來。接下來，我們會知道為什麼要這樣做。
```

Comment:
Here is some duplicate translation in thinking in react part.

Reply:
Yes! Thanks for catching the errors.